### PR TITLE
LibWeb/CSS: Serialize transform scale percentage values as numbers

### PR DESCRIPTION
**Pull request by bplaat**
Fixes http://wpt.live/css/css-transforms/parsing/transform-valid.html :^)

For the reviewer: I have added a Text test which does the same as this wpt test, is this wantend anymore? or is the project going to switch to a wpt testsuite runner soon in CI?

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/StyleValues/TransformationStyleValue.cpp [hunk 1]
```diff
@@ -18,7 +19,22 @@ String TransformationStyleValue::to_string() const
...
+        auto const& value = m_properties.values[i];
+
+        // https://www.w3.org/TR/css-transforms-2/#individual-transforms
+        // A <percentage> is equivalent to a <number>, for example scale: 100% is equivalent to scale: 1.
+        // Numbers are used during serialization of specified and computed values.
+        if ((m_properties.transform_function == CSS::TransformFunction::Scale
+                || m_properties.transform_function == CSS::TransformFunction::Scale3d
+                || m_properties.transform_function == CSS::TransformFunction::ScaleX
+                || m_properties.transform_function == CSS::TransformFunction::ScaleY
+                || m_properties.transform_function == CSS::TransformFunction::ScaleZ)
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/StyleValues/TransformationStyleValue.cpp:31]: Nit-pick: this might be easier to read using `first_is_one_of()`

**Review by AtkinsSJ — APPROVED** [review]
Looks good! One comment you can take or leave.

And yes, tests are still appreciated for now. I don't know how far off WPT CI is.

**Comment by awesomekling** [comment]
> Looks good! One comment you can take or leave.
> 
> And yes, tests are still appreciated for now. I don't know how far off WPT CI is.

WPT CI will never happen, it's takes FAR too long to run, sadly.

**Comment by bplaat** [comment]
> WPT CI will never happen, it's takes FAR too long to run, sadly.

The servo project runs the wpt testsuite separated over multiple runners on each pr, so it is possible

**Comment by awesomekling** [comment]
> > WPT CI will never happen, it's takes FAR too long to run, sadly.
> 
> The servo project runs the wpt testsuite separated over multiple runners on each pr, so it is possible

Fair enough, didn’t know that! Then maybe one day, as long as it isn’t prohibitively expensive :)

**Pull request merged**